applications/GeoMechanicsApplication/custom_utilities/equation_of_motion_utilities.h

```diff
@@ -0,0 +1,37 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Gennady Markelov
+//
+
+#pragma once
+
+// Project includes
+
+// Application includes
+#include "custom_utilities/element_utilities.hpp"
```

Comment:
Minor, but I think this include can move to the cpp file

Reply:
I moved it to the cpp file but I had to add other h files. Nevertheless, a number of dependencies is decreased.